packages/flutter_test/test/live_binding_test.dart

```diff
@@ -16,4 +18,15 @@ void main() {
     // starts with a PointerAddedEvent.
     await gesture.addPointer();
   });
+
+  testWidgets('Input PointerHoverEvent', (WidgetTester tester) async {
+    await tester.pumpWidget(const MaterialApp(home: Text('Test')));
+    await tester.pump();
+    final TestGesture gesture = await tester.createGesture(
+       kind: PointerDeviceKind.mouse);
+    final Offset location = tester.getCenter(find.text('Test'));
+    // for mouse input without a down event, moveTo generates a hover event
+    await gesture.moveTo(location);
```

Comment:
Don't you want to have some expectations after this line?  Or is it just that you expect it to not crash? At the least, you might want to verify that the `Text` actually received a hover event (in which case, you probably want to surround it with a `MouseRegion`).

Reply:
I'm just expecting it not to crash (or rather, not to trigger the assert in the original version before the PR, since that one can actually run on profile mode). The location can be anything for the test purposes but it's just `moveTo` has to have a location.

Reply:
But let me know if you think I should catch the event. @gspencergoog

Reply:
I would suggest at least one expect in there that makes sure that the hover event occurred. Something like:
```dart
  testWidgets('Input PointerHoverEvent', (WidgetTester tester) async {
    PointerHoverEvent hoverEvent;
    await tester.pumpWidget(MaterialApp(home: MouseRegion(child: Text('Test'), onHover: (PointerHoverEvent event){
      hoverEvent = event;
    },)));
    await tester.pumpWidget(const MaterialApp(home: Text('Test')));
    await tester.pump();
    final TestGesture gesture = await tester.createGesture(
        kind: PointerDeviceKind.mouse);
    final Offset location = tester.getCenter(find.text('Test'));
    // for mouse input without a down event, moveTo generates a hover event
    await gesture.moveTo(location);
    expect(hoverEvent, isNotNull);
    await gesture.removePointer();
  });
```

It just makes it clearer that the test does something: without any expectations, someone (like me!) might look at the test and wonder if it is testing anything, or if the expectations were just accidentally omitted.

At the least, you could have a comment saying that the test is expected to not assert (which is probably all you can do for the first test in the file).

Reply:
Done with a little different.